lib/withLoggedInUser.js

```diff
@@ -88,9 +92,11 @@ export default WrappedComponent => {
           throw new Error('Invalid token');
         }
 
-        const newToken = await maybeRefreshAccessToken(token);
+        const newToken = await maybeRefreshAccessToken(token, checkTwoFactorAuth, TOTPCode);
         if (!newToken) {
           throw Error('Invalid token');
+        } else if (newToken === 'twoFactorAuthEnabled') {
+          return { twoFactorAuthEnabled: true };
```

Comment:
Maybe it would be cleaner to throw an Exception, and catch it.

Having `loggedInUser= { twoFactorAuthEnabled: true }` looks too hacky.